src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/SRSConsent/SRSConsentViewModel.swift

```diff
@@ -114,7 +111,7 @@ struct SRSConsentViewModel {
 		ack1.addAttributes(attributes, range: NSRange(location: 0, length: AppStrings.SRSConsentScreen.legalBulletPoint01.count))
 
 		let ack2 = NSMutableAttributedString(string: "\(AppStrings.SRSConsentScreen.legalBulletPoint02)")
-		ack2.addAttributes(attributes, range: NSRange(location: 0, length: AppStrings.SRSConsentScreen.legalBulletPoint02.count))
+		ack2.addAttributes(attributes, range: NSRange(location: 52, length: 36))
```

Comment:
Would be nicer with a calculation, instead of static, but it's okay. Not so important. ;)

Reply:
Will add this to my todo list for later